docker/lisk_service/docker-compose.core.yml

```diff
@@ -25,9 +26,26 @@ services:
     healthcheck:
       test: redis-cli ping
 
+  mysql_core:
+    image: mysql:5.7
```

Comment:
Why MySQL 5 and not 8? Or MariaDB (which we use already)?

Reply:
Node.js libraries have an excellent support for MySQL 5.7. Migration to MySQL 8 is considered in the future.